frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -0,0 +1,356 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Helmet } from 'react-helmet';
+import { match } from 'react-router-dom';
+import { ClusterServiceVersionLogo, iconFor } from './index';
+import {
+  Alert,
+  Bullseye,
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  FlexModifiers,
+  Spinner,
+} from '@patternfly/react-core';
+import { Firehose, FirehoseResourcesResult } from '@console/internal/components/utils';
+import { k8sUpdate, referenceForModel } from '@console/internal/module/k8s';
+import {
+  ClusterServiceVersionModel,
+  InstallPlanModel,
+  PackageManifestModel,
+  SubscriptionModel,
+} from '../models';
+import {
+  CRDDescription,
+  ClusterServiceVersionKind,
+  SubscriptionKind,
+  InstallPlanKind,
+  PackageManifestKind,
+} from '../types';
+import {
+  GreenCheckCircleIcon,
+  RedExclamationCircleIcon,
+  YellowExclamationTriangleIcon,
+} from '@console/shared/src/components/status/icons';
+import { InstallPlanPreview } from './install-plan';
+
+const csvResources = (namespace: string) => [
```

Comment:
Probably something like `const resources = [ ...` as only the first resource here is a CSV.  Or `resourcesForNamespace`, etc.